chassis/invokers/src/main/java/com/griddynamics/jagger/invoker/http/v2/AbstractHttpInvoker.java

```diff
@@ -0,0 +1,58 @@
+package com.griddynamics.jagger.invoker.http.v2;
+
+import com.google.common.base.Preconditions;
+import com.griddynamics.jagger.invoker.InvocationException;
+import com.griddynamics.jagger.invoker.Invoker;
+import com.griddynamics.jagger.invoker.http.ApacheAbstractHttpInvoker;
+import com.griddynamics.jagger.invoker.http.HttpInvoker;
+
+import static java.lang.String.format;
+
+/**
+ * An object that represents abstract HTTP-invoker that invokes services of SuT via http protocol. <p>
+ * Extending classes should provide its own implementation of
+ * {@link #invoke(JHttpQuery query, JHttpEndpoint endpoint)} method.<p>
+ * Also, {@link HTTP_CLIENT httpClient} must be provided by constructor {@link AbstractHttpInvoker#AbstractHttpInvoker(HTTP_CLIENT)}.<br/>
+ *
+ * @param <HTTP_CLIENT> the type of the HTTP-client (look at {@link JHttpClient})<p>
+ * @author Anton Antonenko
+ * @see Invoker
+ * @see ApacheAbstractHttpInvoker
```

Comment:
There is an intention to get rid of that class, so such a reference may mislead.